src/webapp/pages/NeighborhoodFinder.js

```diff
@@ -17,6 +17,9 @@ import sharedStylesFn from "webapp/style/sharedStyles";
 
 const getError = ({ responseError, responseData, str }) => {
   if (responseError) {
+    if (responseError.response.data.message) {
+      return <>{responseError.response.data.message}</>;
```

Comment:
i think you can just return the string here, since its just a string and not a React element.
e.g:
```suggestion
      return responseError.response.data.message;
```

Reply:
Ah, that might have been an errant `npm run fix`. See https://github.com/crownheightsaid/mutual-aid-app/pull/184/commits/53c0de8f62a7b7bbb269ebdd810a17e586faba88 for a fix.